app/views/reports/index.html.slim

```diff
@@ -1,40 +1,88 @@
+              li.tab-title role="presentation" 
+                a aria-controls="annual-eval-fields" data-toggle="tab" href="#annual-eval-fields" role="tab" Annual Evaluations
+              li.tab-title role="presentation" 
+                a aria-controls="acute-rehab-fields" data-toggle="tab" href="#acute-rehab-fields" role="tab" Acute Rehabs
+              li.tab-title role="presentation" 
+                a aria-controls="omr-fields" data-toggle="tab" href="#omr-fields" role="tab"  OMRs
+            /! Configuration tab panes
+            .tabs-content.callout-blank
+              section#patient-fields.content.active role="tabpanel"  
+                - i = 0
+                - for name in Patient.attribute_names
+                  - if i % 3 == 0
+                    .row
+                  - i = i +1
+                  .small-4.columns
+                    = check_box_and_label_tag("export[patient][#{name}]", name)
+              section#annual-eval-fields.content role="tabpanel"  
+                - i = 0
+                - for name in AnnualEvaluation.attribute_names
+                  - if i % 3 == 0
+                    .row
+                  - i = i +1
+                  .small-4.columns
+                    = check_box_and_label_tag("export[annual_evaluation][#{name}]", name)
+              section#acute-rehab-fields.content role="tabpanel"  
+                - i = 0
+                - for name in AcuteRehab.attribute_names
+                  - if i % 3 == 0
+                    .row
+                  - i = i +1
+                  .small-4.columns
+                    = check_box_and_label_tag("export[acute_rehab][#{name}]", name)
+              section#omr-fields.content role="tabpanel"  
+                - i = 0
+                - for name in Omr.attribute_names
+                  - if i % 3 == 0
+                    .row
+                  - i = i +1
+                  .small-4.columns
+                    = check_box_and_label_tag("export[omr][#{name}]", name)
 
     .callout
-     - if results.empty?
-       .row
-         .medium-12.columns
-           p Nothing searched. Nothing found.
-     - else
-       .row
-         .medium-12.columns
-           h4 Download as 
-           a href="#" csv
-           |  
-           a href="#" xls
-         .row
-           .medium-3.columns
-             p First Name:
-           .medium-3.columns
-             p Last Name:
-           .medium-3.columns
-             p Date of Birth:
-           .medium-3.columns
-             p SSN:
-       - for patient in results
-         .row
-           .medium-3.columns
-             = patient.first_name
-           .medium-3.columns
-             = patient.last_name
-           .medium-3.columns
-             = patient.dob
-           .medium-3.columns
-             = patient.ssn
\ No newline at end of file
+      - if results.empty?
+        .row
+          .medium-12.columns
+            p Nothing searched. Nothing found.
+      - else
```

Comment:
This conditional seems to not be working since we are only downloading CSVs. Suggest removing?